Jenkinsfile

```diff
@@ -1,10 +1,16 @@
 elifePipeline {
     node('containers-jenkins-plugin') {
         def commit
+        def grobidTag
+        def fullImageTag
 
         stage 'Checkout', {
             checkout scm
             commit = elifeGitRevision()
+            grobidTag = sh(script: 'bash -c "source .env && echo $GROBID_TAG"', returnStdout: true)
+            echo "GROBID_TAG: ${grobidTag}"
+            fullImageTag = "${grobidTag}-${commit}"
```

Comment:
would put first the part that changes less often, so grobid first followed by the commit seems good